gio/src/subclass/application.rs

```diff
@@ -286,6 +339,8 @@ unsafe impl<T: ApplicationImpl> IsSubclassable<T> for Application {
         klass.shutdown = Some(application_shutdown::<T>);
         klass.startup = Some(application_startup::<T>);
         klass.handle_local_options = Some(application_handle_local_options::<T>);
+        klass.dbus_register = Some(application_dbus_register::<T>);
```

Comment:
please add name_lost as well while you are at it, that would leave only add_platform_data missing which is not really useful as we don't have GVariantBuilder support

Reply:
Okay, will do.